.github/workflows/move2kube-e2e.yaml

```diff
@@ -111,7 +113,9 @@ jobs:
 
       - name: Export kind Logs
         if: always()
-        run: kind export logs ./kind_logs
+        run:
```

Comment:
its not a multiline  run, so the second line it merged into the first
just make it 

```
run: |
```

Reply:
thx